common/src/app/common/pages/common.cljc

```diff
@@ -65,3 +66,320 @@
    :constraints-v         :constraints-group
    :fixed-scroll          :constraints-group})
 
+;; Attributes that may directly be edited by the user with forms
+(def editable-attrs
+  {:frame #{:proportion-lock
```

Comment:
This part of the code is related to the refagor by @hirunatan at https://github.com/penpot/penpot/pull/1540/commits/944a0657b63470ad2a1ff2df6d338d3bf4048a56

Reply:
I thought about this grouping. But the groups are not always exactly the same. For example, if I make a group for "measures" attributes, some shapes have rx, ry, etc. and others not.

Anyway, I'm planning a more general refactor of the shapes specification, that I'd like to do as soon as possible. And in this process it's likely that this could be simplified.